src/utils/utils.js

```diff
@@ -4,8 +4,11 @@
  * @param {Node} target A DOM Node (which may be an Element) within the DOM tree to watch for changes, or to be the root of a subtree of nodes to be watched.
  *
  */
+
+const TIME_OUT = 10000;
```

Comment:
In the `cache.js` file there are consts for `SECOND` and `MINUTE`.

Please move the to the `globals.js` file, then use them here and in `cache.js`.